resource/StandardResources.md

```diff
@@ -0,0 +1,106 @@
+# Standard Resources
+
+This page lists the standard resource types in OpenCensus. For more details on how resources can 
+be combined see [this](Resource.md).
+
+ECS defines these fields.
+ * [Compute Unit](#compute-unit)
+   * [Container](#container)
+ * [Deployment Service](#deployment-service)
+   * [Kubernetes](#kubernetes)
+ * [Compute Instance](#compute-instance)
+   * [Host](#host)
+ * [Environment](#event)
+   * [Cloud](#cloud)
+   * [Cluster](#cluster)
+   * [Data Center](#data-center)
+
+## TODOs
+* Add logical compute units: Service, Task - instance running in a service.
+* Add more compute units: Process, Lambda Function, AppEngine unit, etc.
+* Decide if lower case strings only.
+* Consider to add optional/required for each label.
+
+## Compute Unit
+Resources defining a compute unit (e.g. Container, Process, Lambda Function).
+
+### Container
+**type:** `container`
+
+**Description:** A container instance. This resource can be [merged](Resource.md#Merging) with a
+deployment service resource, a compute instance resource and an environment resource.
+
+| Label  | Description  | Example  |
+|---|---|---|
+| container.id | Unique container id. |  |
+| container.image.name | Name of the image the container was built on. |  |
+| container.image.tag | Container image tag. |  |
+| container.name | Container name. |  |
```

Comment:
Here and below: the intent is to add examples before merging, right?

Reply:
Correct.